tests/integration/test_materialize_mysql_database/materialize_with_ddl.py

```diff
@@ -853,6 +853,16 @@ def materialize_with_column_comments_test(clickhouse_node, mysql_node, service_n
     clickhouse_node.query("DROP DATABASE materialize_with_column_comments_test")
     mysql_node.query("DROP DATABASE materialize_with_column_comments_test")
 
+#def materialize_with_enum_test(clickhouse_node, mysql_node, service_name):
+#    mysql_node.query("DROP DATABASE IF EXISTS materialize_with_enum_test")
+#    clickhouse_node.query("DROP DATABASE IF EXISTS materialize_with_enum_test")
+#    mysql_node.query("CREATE DATABASE materialize_with_enum_test")
+#    mysql_node.query("CREATE TABLE materialize_with_enum_test.test (id int NOT NULL PRIMARY KEY, value ENUM('a','b','c')) ENGINE=InnoDB")
+#    clickhouse_node.query("CREATE DATABASE materialize_with_enum_test ENGINE = MaterializeMySQL('{}:3306', 'materialize_with_enum_test', 'root', 'clickhouse')".format(service_name))
+#    check_query(clickhouse_node, "DESCRIBE TABLE materialize_with_enum_test.test", "id\tInt32\t\t\t\t\t\nvalue\tNullable(String)\t\t\ttest comment\t\t\n_sign\tInt8\tMATERIALIZED\t1\t\t\t\n_version\tUInt64\tMATERIALIZED\t1\t\t\t\n")
+#    clickhouse_node.query("DROP DATABASE materialize_with_enum_test")
+#    mysql_node.query("DROP DATABASE materialize_with_enum_test")
```

Comment:
Please add something to this test with an ALTER TABLE in MySQL that changes the ENUM column to >= 256 values, and insert a row with a 2-byte value.